### ISSUE TITLE
MouseWheel not always returning correct rotation value

### ISSUE THREAD

**Issue by fedeantuna** [TolikPylypchuk/SharpHook#9]
I'm not 100% sure this is a bug, it might be an expected behavior from the base library.

Depending on which window I have active, when I am using a trackpad (on a laptop) I get that the MouseWheel event doesn't return the correct rotation value.
If, for instance, I'm in the "cmd" it just returns 0. The direction works fine but I can't tell if it is left/right or up/down. I am using a Windows machine to test this.

Sample code on .NET 6:

```
using SharpHook.Reactive;

var hook = new SimpleReactiveGlobalHook();

hook.MouseWheel
    .Subscribe(h =>
    {
        Console.WriteLine(h.Data.Direction);
        Console.WriteLine(h.Data.Rotation);
    });

hook.Run();
```

**Comment by TolikPylypchuk** [comment]
Hi! Thanks for posting the issue!

I've just checked and can reproduce the issue - `Rotation` is correct for a mouse, but always `0` for a trackpad. I can't really do anything with this - it has to be fixed in libuiohook. I'll create an issue there.

**Comment by TolikPylypchuk** [comment]
Created [the issue](https://github.com/kwhat/libuiohook/issues/139) in the libuiohook repo.

**Comment by TolikPylypchuk** [comment]
The issue is finally fixed in version 3.1.1.

The problem was that on Windows the rotation value was divided by 120. It's fine for a mouse, but trackpads emit smaller values which means that they were turned into 0. This issue is not yet fixed in libuiohook, so I forked it and fixed it myself.
